flink-docs/src/test/java/org/apache/flink/docs/configuration/ConfigOptionsDocsCompletenessITCase.java

```diff
@@ -61,50 +64,103 @@
 
 	private static final Formatter htmlFormatter = new HtmlFormatter();
 
+	// options for which we allow distinct definitions
+	// this allows reporters to define their own options that are technically only key suffixes
+	private static final Set<String> WELL_DEFINED_WHITELIST = new HashSet<>(Arrays.asList(
```

Comment:
I think we missed `host` and `port` here.

Reply:
yes I did, happened while cleaning up the history...